### PR TITLE
Update usage guidelines in `README.md`.

### PR DESCRIPTION
**Pull request by JSAssassin**
Looking for feedbacks on if the instructions are sufficient/clear for you as a user attempting to create new credentials or contexts, or update existing ones. We want to improve the user experience and ensure that the instructions are easy to follow and helpful. So, any feedback on the clarity and effectiveness of the guidelines are welcome.

### CODE REVIEW

**Review by davidlehn — COMMENTED** [review]
What's here now looks fine.  Potential points of confusion depending on the experience and understanding level of the reader:
- The `safeChef` context does not follow the recommended pattern.  It's a `FoodSafetyCertification`, yet called `safeChef-context-v1.json`.
- It's not that technically important, but someone contributing an example might wonder about using "cleaner" URLs like `https://example.com/foobar/v1`, yet this is saying to use a long server name with a `foobar-context-v1.json` file.  Not sure if it's worth either making that prettier on the server, or just mentioning that it could be improved for a production system?
- Should there be support and a mention of what vocab URLs to use in a context?  It's a bit of a mess right now, and should be cleaned up.  `safeChef` and `movieTicket` contexts are using the 404 `playground.chapi.io`, and `credential-handler.github.io` URLs.  Ideally such URLs could be dereferenced to some doc with resource info.  Where would an example here put such a vocab such that the URLs look nice?

File: README.md [hunk 1]
```diff
@@ -37,15 +37,37 @@ npm test
...
-in the `credential.json` file. Additionally, you can also add a file for the
-credential context in the `./contexts` dir (for example see -
-`./contexts/alumni/v1.json`) if you want to host the contexts in
-[`@credential-handler/vc-playground-contexts-server`](https://github.com/credential-handler/vc-playground-contexts-server).
+### Add / Update a Verifiable Credential
+
+To add or update credentials in the `./credentials` directory:
+
+1. Within the `./credentials` directory, create a new directory for the
+credential. Make sure to use camel case for the directory name.
```
Comment (davidlehn) [inline README.md:45]: #15 started to use [kebab case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case).  The 'introToChapi' dir doesn't follow that pattern.

File: README.md [hunk 2]
```diff
@@ -37,15 +37,37 @@ npm test
...
+3. Additionally, place a logo `image` file for the credential in the
+corresponding directory. The image file should be in either `JPG` or `PNG`
+format.
+
+### Add / Update a Credential Context
+
+To optionally add or update contexts in the `./contexts` directory:
+
+1. If you wish to include a context for a credential, create a new directory
+for the context in the `./contexts` directory. Make sure to use snake case for
```
Comment (davidlehn) [inline README.md:58]: [Snake case](https://en.wikipedia.org/wiki/Letter_case#Snake_case) uses underscores, this should be [kebab case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case).  May want links for these terms.

File: README.md [hunk 3]
```diff
@@ -37,15 +37,37 @@ npm test
...
+2. Please note that once a context is created, it becomes locked and cannot be
+modified. If you want to update or make changes to a context, you must create a
+new context file with a new version within the context directory. For instance,
+if you want to update the context for the "Alumni" credential, create a new file
+named `v2.json` containing the updated JSON-LD context definition within
+`./contexts/alumni`. The previous version,`v1.json`, should be retained in the
+directory without any modifications.
+3. Please note that for the contexts created in the `./contexts` directory, the
+context URL should be of the following format -
+`https://contexts.vcplyaground.org/examples/foo/v1.json`.
```
Comment (davidlehn) [inline README.md:70]: Could be more illustrative for multi word case and use "foo-bar"?

**Comment by davidlehn** [comment]
Does this need to provide guidelines on writing credentials and contexts themselves?  I assume people will copy and modify current examples.  I'm specifically wondering about what to tell people about vocabs to use and where to put custom vocab docs if needed.

File: README.md (hunk 2 above)
Comment (JSAssassin) [reply README.md:58]: Good catch, idk what I was thinking :)

File: README.md (hunk 3 above)
Comment (JSAssassin) [reply README.md]: Updated [5c657a2](https://github.com/credential-handler/vc-examples/pull/15/commits/5c657a20bdb1cb418ebe3ae8ba224138880d912d)

File: README.md (hunk 2 above)
Comment (JSAssassin) [reply README.md]: Fixed [40aaaa2](https://github.com/credential-handler/vc-examples/pull/15/commits/40aaaa29ff59686dcfb3f8336092f8661e6eab42)

File: README.md (hunk 1 above)
Comment (JSAssassin) [reply README.md]: Updated [a707276](https://github.com/credential-handler/vc-examples/pull/16/commits/a707276374dde6b5fadc0ec65d84587cdc9c15d4) and fixed docs [40aaaa2](https://github.com/credential-handler/vc-examples/pull/15/commits/40aaaa29ff59686dcfb3f8336092f8661e6eab42)

**Comment by JSAssassin** [comment]
> Does this need to provide guidelines on writing credentials and contexts themselves? I assume people will copy and modify current examples. 

Not sure i understand this and why copying and modifying current examples could be a problem.

> I'm specifically wondering about what to tell people about vocabs to use and where to put custom vocab docs if needed.

I believe we can ask users to create a `vocab` directory within the `./contexts/foo` dir and include their vocab `index.html` in there if needed?

**Comment by davidlehn** [comment]
> > Does this need to provide guidelines on writing credentials and contexts themselves? I assume people will copy and modify current examples.
> 
> Not sure i understand this and why copying and modifying current examples could be a problem.
> 

It's fine, as long as our examples are perfect best practice examples.  They are, right? :-)
I was just thinking ahead for more complex use cases, but maybe that's not an issue for this system at the moment.


> > I'm specifically wondering about what to tell people about vocabs to use and where to put custom vocab docs if needed.
> 
> I believe we can ask users to create a `vocab` directory within the `./contexts/foo` dir and include their vocab `index.html` in there if needed?

That can work if the server is setup to serve HTML and default to index.html.  I'm not sure if it is?  It's kind of odd to serve other docs from a "contexts" server.  The examples are naming properties with the host name prefix.  In theory, those names should be designed for long term use.  Odd host names are no big deal for one-off examples, but I'd not call that a best practice.

**Comment by JSAssassin** [comment]
@davidlehn 
> It's fine, as long as our examples are perfect best practice examples. They are, right? :-) I was just thinking ahead for more complex use cases, but maybe that's not an issue for this system at the moment.

They are 💯 ;) 
We can point them to VC data model document, hopefully that'll suffice.

> That can work if the server is setup to serve HTML and default to index.html. I'm not sure if it is? 

The server is configured to serve the contents of the 'contexts' directory, which means that anything within that directory will be accessible through the server. I have locally tested this. The HTML files does get rendered successfully.

File: README.md [hunk 4]
```diff
@@ -37,15 +37,45 @@ npm test
...
+2. Please note that once a context is created, it becomes locked and cannot be
+modified. If you want to update or make changes to a context, you must create a
+new context file with a new version within the context directory. For instance,
+if you want to update the context for the "Alumni" credential, create a new file
+named `v2.json` containing the updated JSON-LD context definition within
+`./contexts/alumni`. The previous version,`v1.json`, should be retained in the
+directory without any modifications.
+3. Please make sure that the context URLs for the contexts created in the
+`./contexts` directory follow the format:
+`https://contexts.vcplyaground.org/examples/foo-bar/v1.json`.
```
Comment (ottonomy) [inline README.md:74]: This looks like a typo for the domain name.

Comment (JSAssassin) [reply README.md]: @ottonomy Thank you! Fixed [e4a459f](https://github.com/credential-handler/vc-examples/pull/15/commits/e4a459f7f051efdff6467dbea547438094b69db5)

**Review by ottonomy — APPROVED** [review]
These instructions were easy to follow when I created PR #18.

It may still be a little difficult for people not experienced with JSON-LD to author contexts, but as we collect more examples, the path forward should become pretty well-traveled.

**Review by davidlehn — APPROVED** [review]

**Pull request merged**